app/src/main/java/io/github/sumukhshiv/memberquiz/GameActivity.java

```diff
@@ -0,0 +1,347 @@
+            wrong3 = rand.nextInt(44);
+        }
+        //Get the actual string using random index integer
+        String wrong1Name = imageNames.get(wrong1);
+        String wrong2Name = imageNames.get(wrong2);
+        String wrong3Name = imageNames.get(wrong3);
+
+
+        //id to set the image in the imageView
+        Context c = getApplicationContext();
+        int id = c.getResources().getIdentifier("drawable/"+displayImageName, null, c.getPackageName());
+
+
+        //Gets name from String Array members to get correctly formatted names for all 4 choices
+        for (int i = 0; i < members.length; i++) {
+            if (members[i].replaceAll("\\s+","").equalsIgnoreCase(displayImageName)) {
+                displayImageName = members[i];
+            }
+            if (members[i].replaceAll("\\s+","").equalsIgnoreCase(wrong1Name)) {
+                wrong1Name = members[i];
+            }
+            if (members[i].replaceAll("\\s+","").equalsIgnoreCase(wrong2Name)) {
+                wrong2Name = members[i];
+            }
+            if (members[i].replaceAll("\\s+","").equalsIgnoreCase(wrong3Name)) {
+                wrong3Name = members[i];
+            }
+        }
+
+        //We don't want the correct answer to be in the same location every time
+        //Random number for the location of the correct answer
+        Random randbutton = new Random();
+        int buttonIndex = randbutton.nextInt(4);
+
+        //Array of the wrong button choices' names
+        String[] wrongButtonChoices = new String[4];
+        wrongButtonChoices[0] = wrong1Name;
+        wrongButtonChoices[1] = wrong2Name;
+        wrongButtonChoices[2] = wrong3Name;
+
+        //set the correct button in its location
+        buttonArray[buttonIndex].setText(displayImageName);
+
+        //Add the wrong choices in the reminaing places
+        for (int i = 0,j = 0; i < buttonArray.length && j < wrongButtonChoices.length; i++) {
+            if (buttonArray[i].getText() == "") {
+                buttonArray[i].setText(wrongButtonChoices[j]);
+                j++;
+            }
+        }
+
+        //Set the image of the person
+        image.setImageResource(id);
+
+
+        //DISPLAY COUNTDOWN TIMER
+        timer = new CountDownTimer(5500, 1000) {
+
+            public void onTick(long millisUntilFinished) {
+                textViewTimer.setText("seconds remaining: " + millisUntilFinished / 1000);
+            }
+
+            public void onFinish() {
+                textViewTimer.setText("done!");
+                if (textViewTimer.getText().equals("done!")) {
+                    finish();
+                    startActivity(getIntent());
+                }
+
+            }
+        };
+
+        timer.start();
+
+        //New variable for contact
+        final String contactName = displayImageName;
+
+        //click the image to create a contact
+        image.setOnClickListener(new View.OnClickListener() {
```

Comment:
Weird order, finish what you're doing with the buttons first (or put them in private methods)